src/h5web/vis-packs/core/gammaScale.test.ts

```diff
@@ -0,0 +1,40 @@
+import { gammaScale } from './scales';
+
+describe('gammaScale', () => {
+  it('should have the properties set accordingly', () => {
+    const scale = gammaScale({
+      domain: [-0.5, 0.5],
+      range: [0, 10],
+      exponent: 2,
+    });
+
+    expect(scale.domain()).toEqual([-0.5, 0.5]);
+    expect(scale.range()).toEqual([0, 10]);
+    expect(scale.exponent()).toEqual(2);
+  });
+
+  it('should allow to change properties', () => {
+    const scale = gammaScale({
+      domain: [-10, 10],
+      range: [0, 10],
+      exponent: 2,
+    });
+
+    scale.domain([0, 5]);
+    expect(scale.domain()).toEqual([0, 5]);
+
+    expect(scale(10)).toEqual(10);
```

Comment:
The gamma scale does not behave as a d3-scale. When changing the `domain` via `scale.domain`, the real domain of the scale is not changed even if `scale.domain` returns the new value :disappointed: